dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Ainjector.mo

```diff
@@ -152,6 +152,5 @@ equation
 
   annotation(
     preferredView = "diagram",
-    Icon(graphics = {Rectangle(fillColor = {255, 255, 255}, fillPattern = FillPattern.Solid, extent = {{-100, 100}, {100, -100}}), Text(origin = {0, 30}, extent = {{-90, -30}, {90, 30}}, textString = "IEC WT4A"), Text(origin = {0, -30}, extent = {{-90, -30}, {90, 30}}, textString = "Converter")}, coordinateSystem(initialScale = 0.1)),
-  Diagram);
```

Comment:
same

Reply:
Line 155 has been restored, line 156 is useless.